### DISCUSSION TITLE
Unable to upload file using /v2/files api endpoint

### DISCUSSION THREAD

**Discussion by mfissehaye** [vercel/community#3275] [Help] [answered]
### Summary
![image](https://github.com/vercel/community/assets/2877236/771c0564-c01b-4de4-88fe-add7cc87c6ea)

I am trying to upload a file to vercel api before calling the deployment endpoint. But I keep getting error as mentioned in the image. Need help!

### Steps to Reproduce

1. Call vercel endpoint with the url and headers shown in the postman screenshot attached.

**Comment by mpotrich** [comment, marked as answer]
I was able to do it with these options, using postman
![image](https://github.com/vercel/community/assets/53404250/d99da143-89db-46e8-a8dd-ae08a8506db3)
